### ISSUE TITLE
Simple Import Feeds not importing data

### ISSUE THREAD

**Issue by jpowell53590** [atrocore/atropim#168]
I installed AltroPIM with an empty database.

Now I'm trying to import data into PIM from CSV text files.  But the data is not getting into the database.

Please point me to documentation on how to properly add data to the database.

I have over 1,000 products to add and would not want to add it one by one.

**Comment by rratsun** [comment]
Hello,

We need more information to help. Please, make screenshot of import feed configuration and give an example of file.

**Comment by jpowell53590** [comment]
It will very nice if there was documentation on which tables to populate with data first, second, third.

Or do I need add ALL the table fields, etc in the feed contributor fields?  (meaning more than one table and fields from each table into the feed?

Let me rephrase it in my next comment
